.github/workflows/test_pytorch_wheels.yml

```
@@ -2,6 +2,36 @@ name: Test PyTorch Wheels
 
 on:
   workflow_dispatch:
+    inputs:
+      amdgpu_families:
+        required: true
+        type: string
+        default: "gfx94X-dcgpu"
+      python_version:
+        required: true
+        type: string
+        default: "3.12"
+      s3_bucket:
+        description: "S3 bucket to upload the wheels"
+        required: true
+        type: string
+        default: "therock-nightly-python"
```

The test workflow _downloads_, not _uploads_, right?
```suggestion
      s3_bucket:
        description: "S3 bucket to download wheels from"
        required: true
        type: string
        default: "therock-nightly-python"
```